021_Patcher_L170/021_Patcher_L170_unsafe.kt

```
@@ -0,0 +1,381 @@
+            val archLibFolder = File(apkNativeLibDir, arch)
+            // call callback
+            onIter(arch, archLibFolder)
+        }
+    }
+
+
+    fun AddFileToNativeLibDir(srcFileStr: String) {
+        val srcFile = File(srcFileStr)
+        if (!srcFile.exists()) {
+            throw IOException(String.format("%s doesn't exist",
+                    srcFileStr))
```

detekt (1.23.8)

[warning] String.format("%s doesn't exist",
                    srcFileStr) uses implicitly default locale for string formatting.

(detekt.potential-bugs.ImplicitDefaultLocale)